source/javascripts/components/ProductTour/useHighlightedArea.ts

```diff
@@ -41,7 +41,7 @@ export const useHighlightedArea = (selectedId: string | undefined): DOMRect | nu
 	useEffect(() => {
 		const onResize = (): void => {
 			if (ref.current) {
-				const rect = ref.current!.getBoundingClientRect();
+				const rect = ref.current.getBoundingClientRect();
```

Comment:
I didn't receive any errors about it, likely because line 43 already has a check in place for the existence of the ref. Initially I made the update because I thought it was the root cause of the bug I was talking about in standup, but now that I know it isn't, I've reverted it back to the prior value.